src/components/Authorization.js

```diff
@@ -0,0 +1,198 @@
+import React, { Component, createContext } from 'react';
+import { withRouter } from "react-router-dom";
+import createStoreAccess from '../utils/createStoreAccess'
+
+const AuthorizationContext = createContext(null)
+
+class Authorization extends Component {
+    // Store key names.
+    static tokenStorageName = 'token'
+    static userStorageName = 'user'
+
+    // Query params passed from the API during login.
+    static authTokenParamName = 'token'
```

Comment:
again token? Not sure if correct when in line 9 is also 'token'

Reply:
Token in the 9th line refers to the local storage key. This one is a name of the param coming from backend. Their equality is coincidental.